charter-draft.html

```diff
@@ -47,7 +47,11 @@ <h1 id="title">
 				DRAFT Privacy Interest Group Charter
 			</h1>			
 			<p class="mission">
-				The <strong>mission</strong> of the <a href="https://www.w3.org/Privacy/">Privacy Interest Group</a> is to improve the support of privacy in Web standards by monitoring ongoing privacy issues that affect the Web, investigating potential areas for new privacy work, and providing guidelines and advice for addressing privacy in standards development.
+				The <strong>mission</strong> of W3C's <a href="https://www.w3.org/Privacy/">Privacy Interest Group</a>
+				(PING) is to improve privacy on the Web by advising groups
+				developing standards on how to avoid and mitigate privacy issues with their
+				technologies and by incubating new work that reduces fingerprinting surface area
```

Comment:
debating whether fingerprinting is overly specific for "mission"...not going to fight it, as it's something that comes up a lot, but it does make it seem very central if it's in the mission!